src/ui/components/shared/LoadingTip.tsx

```diff
@@ -1,7 +1,7 @@
 import classNames from "classnames";
 import React, { useEffect, useRef } from "react";
 import { connect, ConnectedProps } from "react-redux";
-import { actions } from "ui/actions";
+import { setLoadingPageTipIndex } from "ui/actions/app";
```

Comment:
this fixed storybook 🤦  In the future, we could look in webpack's bootstrap function to find where webpack failed